tests/ci/test_docs_end_to_end/setup_test.sh

```
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+echo "Running python3 main.py --all-servers"
+cd ${AIPERF_SOURCE_DIR}/tests/ci/${CI_JOB_NAME}/
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)